examples/vic-gov-au/assets/ripple-icon/icon_circle_arrow_right.js

```diff
@@ -0,0 +1,13 @@
+import Icon from '../../../../packages/components/Atoms/Icon/Icon'
+
+Icon.register({
+  'icon_circle_arrow_right': {
+    width: 10,
+    height: 10,
+    paths: [
+      {
+        d: 'M0 10c0 5.524 4.476 10 10 10s10-4.476 10-10S15.524 0 10 0 0 4.476 0 10zm18.065 0A8.062 8.062 0 0110 18.065 8.062 8.062 0 011.935 10 8.062 8.062 0 0110 1.935 8.062 8.062 0 0118.065 10zM10.44 4.762l4.895 4.895c.19.19.19.496 0 .686l-4.895 4.895a.484.484 0 01-.686 0l-.79-.79a.483.483 0 01.008-.694l2.834-2.706H5.323a.485.485 0 01-.484-.483v-1.13c0-.266.217-.483.484-.483h6.483L8.972 6.246a.483.483 0 01-.008-.694l.79-.79c.19-.19.496-.19.686 0z'
+      }
+    ]
+  }
+})
```

Comment:
This will be how to register custom icons/if they want to replace Ripple icon with their own they just have to register it with the same name as in Ripple.